unity-renderer/Assets/DCLPlugins/ECS7/InternalECSComponents/Interfaces/Extensions/InternalRenderersExtensions.cs

```diff
@@ -18,15 +18,15 @@ public static class InternalRenderersExtensions
             if (model == null)
                 return;
 
-            model.renderers.Remove(renderer);
+            model.Value.renderers.Remove(renderer);
 
-            if (model.renderers.Count == 0)
+            if (model.Value.renderers.Count == 0)
             {
-                renderersInternalComponent.RemoveFor(scene, entity, new InternalRenderers());
+                renderersInternalComponent.RemoveFor(scene, entity, new InternalRenderers(new List<Renderer>()));
```

Comment:
here as well - is it possible to cache, clear and re-use instead of creating new instances?